public/app/fn-app/create-mfe.ts

```diff
@@ -246,17 +246,26 @@ class createMfe {
       createMfe.logger('Trying to update grafana with theme:', props.mode);
 
       if (props.mode) {
+        dispatch(
```

Comment:
This dispatch function is not updating anything.

Reply:
@gurinder39 

**It is dispatching.**

1. I  added log in the  'updateFnState' reducer
2. The 'updateFnState' is called only in one place in the app (in the commented line)
3. SO, if the log in the reducer happens it means that the action is dispatched
4. Last but not least, the log happens (see mp4 and screenshots).

Conclusion: the action is dispatched.

![Screenshot from 2022-10-21 17-06-19](https://user-images.githubusercontent.com/24753130/197228145-538daa48-bbbb-4ef7-9697-8d0b48f5d578.png)



https://user-images.githubusercontent.com/24753130/197227875-b4b876df-5e13-4f21-9ca0-dfad1b767ca3.mp4